File: OpenValidationFramework-Python/OpenValidation.py
import HUMLFramework as huml

class HUMLValidator(huml.HUMLFramework):
    def __init__(self):
        super().__init__()



        self.append_rule(
            "",
            ["number"],
            "number must be greater than 5",
            lambda model, cntx: huml.LESS_OR_EQUALS(model.number, 5.0),
            False
        )

File: OpenValidationFramework-Javascript/OpenValidation.js
var HUMLValidator = function() {
    var huml = new HUMLFramework();

            huml.appendRule("",
                   ["Name"],
                   "Name MUST be Jerry",
                   function(model) { return huml.NOT_EQUALS(model.Name, "Jerry");},
                   false
                );

    

    this.validate = function(model){
        return huml.validate(model);
    }
}


